larpandoracontent/LArMonitoring/TwoViewTransverseTracksValidationTool.h

```diff
@@ -0,0 +1,237 @@
+/**
+ *  @file   larpandoracontent/LArMonitoring/TwoViewTransverseTracksValidationTool.h
+ *
+ *  @brief  Header file for the two view transverse tracks validation tool
+ *
+ *  $Log: $
+ */
+#ifndef TWO_VIEW_TRANSVERSE_TRACKS_VALIDATION_TOOL_H
+#define TWO_VIEW_TRANSVERSE_TRACKS_VALIDATION_TOOL_H 1
+
+#include "PandoraMonitoringApi.h"
```

Comment:
Will this (and maybe some other places) need wrapping in an `#ifdef MONITORING`?

Since its possible to compile Pandora without monitoring support, I would assume so, though I believe all the macro usage is fine. Could be worth compiling in standalone without it to check maybe?

Reply:
Good spot, yes they will.

Reply:
Got it.  TY Ryan.
As far as I can tell, I only need to the protection for including the `PandoraMonitoringApi.h` header as, elsewhere, I only talk to the API via the provided C macro: (`PANDORA_MONITORING_API`).  Am I correct here?

Reply:
Anything enclosed by `PANDORA_MONITORING_API` will reduce to no code if Monitoring has not been enabled in the build. Still typically want to test a no-monitoring build, seeing as we have -wUnused and -wError as build flags, and can often have end up with unused variables in the monitoring-off situation.